landlab/components/detachment_ltd_erosion/generate_detachment_ltd_erosion.py

```diff
@@ -161,26 +160,24 @@ def __init__(
             changes in topographic elevation due to tectonic uplift
         entrainment_threshold : float, optional
             threshold for sediment movement
+        slope : str
+            Field name of an at-node field that contains the slope.
         """
-        super(DetachmentLtdErosion, self).__init__(grid, **kwds)
+        super(DetachmentLtdErosion, self).__init__(grid)
 
-        self.K = K_sp
-        self.m = m_sp
-        self.n = n_sp
+        assert slope in grid.at_node
 
-        self.I = self._grid.zeros(at="node")  # noqa: E741
-        self.uplift_rate = uplift_rate
-        self.entrainment_threshold = entrainment_threshold
+        self._K = K_sp
+        self._m = m_sp
+        self._n = n_sp
 
-        self.dzdt = self._grid.zeros(at="node")
+        self._I = self._grid.zeros(at="node")  # noqa: E741
```

Comment:
This component previously used a public attribute called `self.I`. I changed it to `self._I`.

Reply:
Sorry, I wasn't clear, but I looked up what "noqa" means

Reply:
@gregtucker the `# noqa: E741` was to disable the linter from warning about using a variable named *I* (because it can easily be mistaken with a lowercase *l* in many text editors). Because of the underscore, we may no longer strictly need the pragma but, even still, ideally we should come up with a better name. It's just asking for a typo in the future!

Reply:
@gregtucker @kbarnhart Oh, sorry, I see what your question was now. How is this
```python
self.dzdt = self._grid.zeros(at="node")
```
equivalent to this
```python
self._I = self._grid.zeros(at="node")
```
Why not, instead,
```python
self._dzdt = self._grid.zeros(at="node")
```
Was that what your *?* was about?

Reply:
![Screen Shot 2019-09-27 at 11 25 35 AM](https://user-images.githubusercontent.com/6589339/65788982-96168100-e119-11e9-8906-9f64a0b1b606.png)

@mcflugen see screenshot... I think your question has more to do with GitHub's like comparison than any change I made.

Reply:
Sorry for confusion, I just hadn't known about the # noqa directive